lua/lls.lua

```diff
@@ -7,7 +7,10 @@ return function (net_conf, numa_table)
 	end
 
 	-- Change these parameters to configure the LLS block.
-	lls_conf.debug = false
+
+	-- Log level for LLS. Note: set to RTE_LOG_DEBUG
+	-- to output routing table updates to the log.
+	lls_conf.log_level = gatekeeper.c.RTE_LOG_DEBUG
```

Comment:
Complement the second line of the comment above as follows:
```
-- to log routing table updates and to log all the cache when it is periodically scanned.
```
Break line as needed. I used the old comment of the dropped parameter `debug` to complement the comment above.